src/main/java/com/coveros/selenified/element/WaitFor.java

```diff
@@ -58,7 +58,7 @@
     private static final String ENABLED = " to be enabled";
 
     private double defaultWait = 5.0;
-    private long defaultPollingInterval = 50;
+    private final long defaultPollingInterval = 50;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1170)